api/handlers.go

```diff
@@ -40,6 +40,15 @@ var (
 	// flow fails. This error is sent instead of whatever internal error we had
 	// before in order to prevent an attacker from listing our users.
 	ErrInvalidCredentials = errors.New("invalid credentials")
+
+	// MyskyAllowlist contains skylinks we need to make available in order for
+	// users to be able to use MySky on all portals, including ones that require
+	// user authentication.
+	MyskyAllowlist = map[string]interface{}{
```

Comment:
Is this something that might eventually live in mongo? Obviously, I'm not suggesting consulting mongo on every call but the whitelist could get reloaded every 5' in the background or whatever. These skylinks are pretty static so I guess hardcoding them is fine but at the same time, it might be a useful feature to grant full access to certain skylinks for everyone. Could turn into a pretty cool feature 👍

Reply:
If we're talking about a general allowlist that's in the control of the operator, that makes sense and it could turn into a cool feature if we come up with a nice user story. But here we have a rather special case where we, as the devs of the portal, force a set of skylinks to be accessible on all portals without the portal operator having any say in that. While we have a pretty solid reason to do this, I really hope that this will remain a strictly isolated incident and this list will never grow.